README.md

```diff
@@ -1,3 +1,8 @@
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.javacc/javacc-core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.javacc/javacc-core)
```

Comment:
Are you sure it's the proper URL? It seems the URL is for javacc not for javacc21?

Reply:
javacc21 is com.javacc

Reply:
Ok thanks for the info.